src/ty.md

```diff
@@ -2,94 +2,96 @@
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Describe the *syntax* of a type: what the user wrote (with some desugaring).  | Describe the *semantics* of a type: the meaning of what the user wrote. |
-| Each `rustc_hir::Ty` has its own spans corresponding to the appropriate place in the program. | Doesn’t correspond to a single place in the user’s program. |
-| `rustc_hir::Ty` has generics and lifetimes; however, some of those lifetimes are special markers like [`LifetimeName::Implicit`][implicit]. | `ty::Ty` has the full type, including generics and lifetimes, even if the user left them out |
-| `fn foo(x: u32) → u32 { }` - Two `rustc_hir::Ty` representing each usage of `u32`. Each has its own `Span`s, etc.- `rustc_hir::Ty` doesn’t tell us that both are the same type | `fn foo(x: u32) → u32 { }` - One `ty::Ty` for all instances of `u32` throughout the program.- `ty::Ty` tells us that both usages of `u32` mean the same type. |
-| `fn foo(x: &u32) -> &u32)`- Two `rustc_hir::Ty` again.- Lifetimes for the references show up in the `rustc_hir::Ty`s using a special marker, [`LifetimeName::Implicit`][implicit]. | `fn foo(x: &u32) -> &u32)`- A single `ty::Ty`.- The `ty::Ty` has the hidden lifetime param |
+**Order** 
 
-[implicit]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/enum.LifetimeName.html#variant.Implicit
+[HIR](hir.md) is built off the [AST](syntax-intro.md) before any [`ty::Ty`] is
+produced. After [HIR](hir.md) some type inference and type checking is done.
+During type inference a [`ty::Ty`] for every element is determined and
+ambiguities are identified. Now [`ty::Ty`] can be checked to confirm typing
+information.
 
-**Order** HIR is built directly from the AST, so it happens before any `ty::Ty` is produced. After
-HIR is built, some basic type inference and type checking is done. During the type inference, we
-figure out what the `ty::Ty` of everything is and we also check if the type of something is
-ambiguous. The `ty::Ty` then, is used for type checking while making sure everything has the
-expected type. The [`astconv` module][astconv] is where the code responsible for converting a
-`rustc_hir::Ty` into a `ty::Ty` is located. This occurs during the type-checking phase,
-but also in other parts of the compiler that want to ask questions like "what argument types does
-this function expect?"
+The [`astconv`][astconv] module handles the actual conversion of a
+[`rustc_hir::Ty`][hir_ty] into a [`ty::Ty`] during the type-checking phase, or
+in other parts of the compiler which need specific type-based information such
+as (but not limited to): "what argument types does this function expect?"
```

Comment:
nit: "but not limited to" isn't necessary here. "such as" implies there are other questions.